### PR TITLE
[FLINK-26158] Update the playground examples to use the playground ingress/egress

### PR DESCRIPTION
**Pull request by tillrohrmann**
This PR is based on #25. It updates the playground to use the newly introduced playground ingress/egress. By using the curl based ingress/egresses we can simplify the overall setup and get rid of some helper functions.

### CODE REVIEW

File: java/greeter/docker-compose.yml [hunk 1]
```diff
@@ -35,51 +35,12 @@ services:
   ###############################################################
 
   statefun:
-    image: apache/flink-statefun-playground:3.2.0
+    image: apache/flink-statefun-playground:3.2.0-1.0
```
Comment (igalshilman) [inline java/greeter/docker-compose.yml:38]: I don't think it is needed, you can republish the same docker image (`apache/flink-statefun-playground:3.2.0`)

Comment (tillrohrmann) [reply java/greeter/docker-compose.yml:38]: Oh really? So it would only be a problem if it is cached somewhere, right?

**Comment by tillrohrmann** [comment]
Thanks for the quick review @igalshilman :-)

**Comment by tillrohrmann** [comment]
Merging this PR now.

**Pull request merged**